lib/lazy_high_charts/layout_helper.rb

```diff
@@ -81,8 +81,8 @@ def request_is_referrer?
       defined?(request) && request.respond_to?(:headers) && request.headers["X-XHR-Referer"]
     end
 
-    def request_turbolinks_5_tureferrer?
-      defined?(request) && request.respond_to?(:headers) && request.headers["Turbolinks-Referrer"]
+    def is_turbolinks_5?
+      Gem::Version.new(Turbolinks::VERSION) >= Gem::Version.new('5.0.0')
```

Comment:
Interesting approach. Question: what would happen if you have Turbolinks present in the project but not enabled in javascript ? I think in that case this solution will always return true

Reply:
I agree with @michelson ... The solution that only checks the gem version causes problems when I'm rendering charts in a PDF, where I have JS enabled but Turbolinks is not enabled (it is on my application though)

Reply:
that has been fixed long ago with this [this](https://github.com/michelson/lazy_high_charts/pull/224/commits/538525468ca96b1662a9316cd540d46790419cf7) commit

Reply:
Well, my charts on PDFs are still not being rendered, don't know why... (they are rendered normally when using @michelsons master).